CHANGELOG.md

```diff
@@ -93,6 +93,7 @@
 
 ## [unreleased]
 
+- When creating an activity, the `Tied status` step has been removed from the user journey and it has now a default value of `Untied`, code "5"
```

Comment:
I think, by convention, we always add the new change to the bottom of the list.

Reply:
Yes. That was my mistake when rebasing. Thanks for pointing that out. I've fixed it now.